userapi/storage/accounts/sqlite3/storage.go

```diff
@@ -41,13 +41,15 @@ type Database struct {
 	profiles     profilesStatements
 	accountDatas accountDataStatements
 	threepids    threepidStatements
+	openIDTokens tokenStatements
 	serverName   gomatrixserverlib.ServerName
 	bcryptCost   int
 
 	accountsMu     sync.Mutex
 	profilesMu     sync.Mutex
 	accountDatasMu sync.Mutex
 	threepidsMu    sync.Mutex
+	openIDsMu      sync.Mutex
```

Comment:
You shouldn't need a mutex for this. I _think_ you only need mutexes for tables which are modified on registration/login.

Reply:
Yeah, this only ever gets written to. The mutex was not needed.